os/binfmt/binfmt_unloadmodule.c

```diff
@@ -64,6 +64,7 @@
 
 #include <tinyara/kmalloc.h>
 #include <tinyara/binfmt/binfmt.h>
+#include <tinyara/mpu.h>
```

Comment:
Could you add conditions for mpu, `#if defined(CONFIG_SUPPORT_COMMON_BINARY) && defined(CONFIG_ARCH_MPU)`?